capability_map:
  resolution: 0.01
  search_factor: 1.0
  group_name: "l_arm"
  # Database config
  database:
    server: "localhost"
    port: 27017
    name: "workspace_analysis"
    collection: "capability_map"


